src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,357 @@
+
+The number of networked Beacon Chain nodes (i.e. nodes running libp2p) is [around 5,000 at the current count](https://migalabs.es/beaconnodes) with this [client diversity](https://clientdiversity.org/#distribution).
+
+<div class="container" style="display:flex; column-gap:10px;">
+    <figure>
+        <img src="../assets/eth-libp2p-beacon-nodes.png" width="600">
+        <figcaption style="font-size:x-small;">
+            <a href="https://migalabs.es/beaconnodes">[Miga Labs dashboard.]</a>
+        </figcaption>
+    </figure>
+</div>
+
+libp2p is running on all those consensus clients; a fantastic testament to 6+ years of hard work!
+
+Additionally, there are over [495,000 validators](https://beaconcha.in/validators) amongst these nodes (validators are [virtual entities](https://launchpad.ethereum.org/en/faq) that live on the Beacon Chain.)
+
+## What's next? :rocket:
+
+Now that the Merge has gone smoothly, the primary collaboration efforts have been resolved successfully.
+At this time, Protocol Labs is supporting the Ethereum community in ensuring the libp2p is running smoothly and reliably.
+Some of the future roadmap items for libp2p and Ethereum have been hinted at in the specification (like exploring the usage of QUIC).
+When the time comes for that, the collaboration will pick up steam once more!
+
+In conclusion, the libp2p and Ethereum integration has been a massive effort.
+It could not have been possible without the grit and perseverence of many organizations and invididuals.
+Thank you to everyone invovled, and onwards to our next goals!
+
+We hope you enjoyed our summary of the journey and hope you learned a few things along the way.
+Thank you for reading! :pray:
+
+### Resources and how you can help contribute 💪
+
+If you would like to learn more about libp2p, please see the libp2p:
+
+- [Docs](https://docs.libp2p.io/)
+- [Connectivity Website](https://connectivity.libp2p.io/)
+    - This describes the various libp2p transport implementations
+- [libp2p Specifications](https://github.com/libp2p/specs/)
+
+If you would like to contribute, please [connect with the libp2p maintainers](https://libp2p.io/#community).
+
+---
+## Comparing devp2p and libp2p 🤼
+
+This is an optional section that describes the differences between devp2p and libp2p.
+The [devp2p repo](https://github.com/ethereum/devp2p#relationship-with-libp2p) provides an apt contrast of each project's intended scope and design:
+
+*"devp2p is an integrated system definition that wants to serve Ethereum's needs well (although it may be a good fit for other applications, too) while libp2p is a collection of programming library parts serving no single application in particular.
+That said, both projects are very similar in spirit and devp2p is slowly adopting parts of libp2p as they mature."*
+
+Meaning devp2p was explicitly designed to fulfill requirements for Ethereum (more on how devp2p adopted parts of libp2p [in the next section](#Coverging-devp2p-and-libp2p-)).
```

Comment:
I don't see a section for "#Coverging-devp2p-and-libp2p-"

Reply:
Deleted it now, thanks for the catch, it was an artifact of another section I ended up removing.